assets/stylesheets/host.css

```diff
@@ -9,4 +10,229 @@ body {
 .frame {
   height: 100%;
   width:  100%;
+  margin-top: 15px;
+}
+
+.modal-container {
+  animation: modalContainerFade ease 0.5s;
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  left: 0;
+  top: 0;
+  background-color: rgba(0, 0, 0, 0.5);
+  z-index: 1;
+}
+
+@keyframes modalContainerFade {
+  0% {
+    background-color: rgba(0, 0, 0, 0);
+  }
+  100% {
+    background-color: rgba(0, 0, 0, 0.5);
+   }
+}
+
+.modal {
+  animation: modalFade ease 0.5s;
+  background-color: white;
+  position: relative;
+  top: 50%;
+  transform: translateY(-50%);
+  margin-left: auto;
+  margin-right: auto;
+  padding-bottom: 15px;
+  width: 1000px;
+  height: 550px;
+  border: 5px solid;
+  border-radius: 20px;
+}
+
+@keyframes modalFade {
+  0% {
+   opacity: 0;
+  }
+  100% {
+    opacity: 100%;
+   }
+}
+
+.modal-text {
+  width: 950px;
+  height: 450px;
+  margin: 25px 25px 0px 25px;
+  overflow: scroll;
```

Comment:
For whatever reason, in Chromium, I'm seeing permanent scroll bars on this element, both horizontally and vertically.  When I remove this line (`overflow: scroll`), they go away.

Reply:
Which isn't to say that I think this element shouldn't scroll.  It's just that something is funky about how Chromium is understanding the styling/sizing of this element.

Reply:
Just talked about this & resolved.

Reply:
We determined the solution here to be using `overflow-y: auto` instead.  MacOS hides scrollbars with `overflow: scroll`, despite them showing up on other platforms.